### PR TITLE
making related library cards look just like user profile library cards

### PR DESCRIPTION
**Pull request by 2is10**
@yipingliao @joshm1 

Also removing hard-coding of colors, since Aaron has populated colors for every library and one is randomly chosen for every newly created library now.

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/libraries/relatedLibraries.styl [hunk 1]
```diff
@@ -83,16 +84,21 @@ mobileBorderRadius = 54px
...
-  margin-bottom: 1em
+  margin-bottom: 2em
   .kf-mobile &
-    margin: 40px 25px 10px 30px
+    margin: 0 auto 500px
     float: none
-    width: auto
+    width: 210px
+    transform: scale(2.5)
+    transform-origin: center top
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/libraries/relatedLibraries.styl:98]: using big margin-bottom + scaling to get accurate larger rendering on mobile

**Comment by 2is10** [comment]
I started by making small changes to the small library card widget to make its appearance more similar, then decided it would be faster to just replace the whole template and all styles with those from the user profile. Just using a different class prefix.

**Comment by 2is10** [comment]
This may still need tweaking. Merging now to get it out for evaluation and QA. Happy to address any comments in a follow-up.

File: kifi-backend/modules/shoebox/angular/src/libraries/smallLibraryCard.styl [hunk 2]
```diff
@@ -1,198 +1,174 @@
...
+  border-radius: 16px
+  vertical-align: top
   position: relative
-  border-radius: desktopBorderRadius desktopBorderRadius
-  z-index: 1
+  overflow: hidden
   text-align: center
-  color: #fff
-  font-family: Lato
+  box-shadow: 0 3px #f5f5f5, 0 4px #c9ccc0, 0 6px #e5e5e5, 0 7px #c4c6cb, 0 8px #d4d4d4, 0 9px #bec1c5
```
Comment (joshm1) [inline kifi-backend/modules/shoebox/angular/src/libraries/smallLibraryCard.styl:11]: nice! didn't know you could do this

**Comment by joshm1** [comment]
lgtm, thanks!

**Pull request merged**